index.html

```diff
@@ -1,278 +1,347 @@
+                </h4>
+                <ul class="card-tags">
+                    <li>React</li>
+                    <li>Apollo-client</li>
+                    <li>Apollo-server</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="main.png" alt="img">
+                </div>
+                <h4 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h4>
+                <ul class="card-tags">
+                    <li>React</li>
+                    <li>Firebase</li>
+                    <li>JavaScript</li>
+                    <li>Css</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="main.png" alt="img">
+                </div>
+                <h4 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h4>
+                <ul class="card-tags">
+                    <li>Rubi on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="main.png" alt="img">
+                </div>
+                <h4 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h4>
+                <ul class="card-tags">
+                    <li>Rubi on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="main.png" alt="img">
+                </div>
+                <h4 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h4>
+                <ul class="card-tags">
+                    <li>Rubi on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button" class="btn-popup">
+                    See Project
+                </button>
+            </article>
+        </div>
+    </section>
+    <section class="container-flex about-myself" id="about-myself-id">
```

Comment:
- [ ] While doing the **desktop version**, you mismatch the **mobile version** of the `About me` section. Kindly fix that.
- Your website

![sdvdv](https://user-images.githubusercontent.com/26767383/155520356-52076e36-4e12-4878-b72a-e6afa65f2c34.PNG)

- Figma website

![dbfdfbd](https://user-images.githubusercontent.com/26767383/155520403-3d24f825-e8b2-496c-9728-a9420f555fcb.PNG)